pkg/trace/traceutil/span.go

```diff
@@ -40,6 +42,15 @@ func IsMeasured(s *pb.Span) bool {
 	return s.Metrics[measuredKey] == 1
 }
 
+// IsPartialSnapshot returns true if the span is a partial snapshot.
+// This kind of spans are partial images of long-running spans.
+// When incomplete, a partial snapshot has a metric _dd.partial_version which is a positive integer.
+// The metric usually increases each time a new version of the same span is sent by the tracer
+func IsPartialSnapshot(s *pb.Span) bool {
+	v, ok := s.Metrics[partialVersionKey]
+	return ok && v >= 0
```

Comment:
Looking at this PR much later in hindsight, shouldn't `v` be strictly greater than 0?

Reply:
🤔 Makes sense. @amarziali ?

Reply:
that's true that we decided to have this value be starting with 1 but it's not a big issue to support >=0. `0` has no special meaning. The value is intended to be monotonic increasing. It can be safely changed to `>0` if wished